pipelines/common/src/main/java/com/google/fhir/analytics/DwhFiles.java

```diff
@@ -188,15 +188,18 @@ public boolean hasIncrementalDir() throws IOException {
 
   /**
    * This method returns the list of non-empty directories which contains at least one file under
-   * it. The directory name should be a valid FHIR resource type.
+   * it.
    *
+   * @param viewFlag If true, the views returned will be valid FHIR Resource Types. If false, the
```

Comment:
Then I think the name of this param should be something like `findResourceTypes`.

Reply:
Done.